build-support/bin/travis-ci.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
```

Comment:
A comment in here explaining what should go in this script ("setup specific to the python environment present on travis") would be good.

Reply:
Agreed - fix coming.

Reply:
Fixed. This will probably required one more rebase to pull in #6446 to get CI green.